addons/gamification/tests/test_challenge.py

```diff
@@ -184,6 +184,62 @@ def test_30_create_challenge_with_sum_goal(self):
             "Challenge failed to start",
         )
 
+    def test_40_create_challenge_with_sum_goal(self):
+        challenge = self.env['gamification.challenge'].create({
+            'name': 'test',
+            'state': 'draft',
+            'user_domain': '[("active", "=", True)]',
+            'reward_id': 1,
+        })
+
+        Goals = self.env['gamification.goal']
+
+        model = self.env['ir.model'].search([('model', '=', 'res.company')])[0]
```

Comment:
It's a bit unrealistic to use res.company here, why not an all-users available one?